web-new/src/components/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+import Image from "next/image";
+import logoDarklake from "../../public/images/logo-h-darklake.png";
+
+const Header = () => {
```

Comment:
Minor, non-blocking, I think we could just as easily define these as `function Header() { ... }`, might be cleaner